docs/reference/sdk-guidelines.mdx

```diff
@@ -0,0 +1,679 @@
+<br />
+<div align="center">
+  <a href="">
+    <img src="./" alt="Logo" width="300" height="auto">
+  </a>
+  <p align="center"> <br />
+    <a href=""><strong>View on GitHub »</strong></a> <br /><br />
+    <a href="https://docs.onflow.org/sdk-guidelines/">SDK Specifications</a> ·
+    <a href="">Contribute</a> ·
+    <a href="">Report a Bug</a>
+  </p>
+</div><br />
+
+## Overview 
+
+This reference documents all the methods available in the SDK, and explains in detail how these methods work.
+SDKs are open source, and you can use them according to the licence.
+
+The library client specifications can be found here:
+
+// TODO specs here
+[<img src="https://raw.githubusercontent.com/onflow/sdks/main/templates/documentation/ref.svg" width="130">]()
+
+
+## Getting Started
+
+### Installing
+NPM:
+```
+npm install --save @onflow/fcl @onflow/types
+```
+
+Yarn:
+```
+yarn add @onflow/fcl @onflow/types
+```
+
+### Importing the Library
+```
+import * as fcl from "@onflow/fcl"
+import * as types from "@onflow/types"
+```
+
```

Comment:
feel free to add text specific to js SDK (like couple of lines on explaining fcl/sdk could be useful)